### PR TITLE
Updated the Project Code

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Updated the Read Me File

### CODE REVIEW

**Comment by AsimKhan2019** [comment]
Leaving the request open for review

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no review from you. You are supposed to review your work like a code reviewer does.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (AsimKhan2019) [inline README.md:2]: - [ ] Nicely done putting in the link for review.

**Comment by AsimKhan2019** [comment]
@Munsa1 Thank you for the guideline. I have just added some reviews to the Read Me File.

**Comment by AsimKhan2019** [comment]
@Munsa1 Please tell me if this is ok.

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019    👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
 - Please add Proffessional and descriptive PULL REQUEST 
- Please use github inline comment to add reviews and changes to you past project from the trial. In short please review your project and add some comments on what you didnt do correctly. Kindly read through the project requirement properly for a better understanding. Thanks 
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by AsimKhan2019** [comment]
@KayLemba Thank you for the guidelines. I wanted some clarity, I have added inline reviews to the code. Can you please provide some more clarity? As for the trials, this is the project I did in the past and made some adjustments to the project. Kindly please provide some more guidelines.

**Comment by KayLemba** [comment]
There is a lesson on this in the material provided by microverse please kindly go through it. check your notes

**Comment by KayLemba** [comment]
You need to give yourself a review

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h3>Youtube Video</h3>
       <section id="content">
         <iframe  id="frameVideo" width="560" height="315" src="https://www.youtube.com/embed/5gFGX43vubM" 
               title="YouTube video player" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
-              allowfullscreen></iframe>
+              allowfullscreen></iframe> <!-- removed frameborder -->
```
Comment (AsimKhan2019) [inline index.html:20]: - [ ]  You have applied the right approach here. The frame border attribute is obsolete.

**Comment by AsimKhan2019** [comment]
@KayLemba I think I understand what you mean. Can you please look at the repository now, and see if this is correct?

File: style.css [hunk 3]
```diff
@@ -14,7 +14,7 @@ body {
 
 #header {
   display: flex;
-  font-size: medium;
+  font-size: large;
   font-family: Arial, Helvetica, sans-serif;
   font-weight: bold;
   padding-left: 25px;
```
Comment (AsimKhan2019) [inline style.css:20]: - [ ] You have added in the header tag element, which is a good practice. Let's use numeric pixel values for font size instead, that is much better.

**Comment by AsimKhan2019** [comment]
@KayLemba How about now? I have gone through the tutorial. I have made some adjustments to the code review. Please let me know your thoughts.

**Comment by ishimwezachee** [comment]
> @KayLemba How about now? I have gone through the tutorial. I have made some adjustments to the code review. Please let me know your thoughts.

Hello @AsimKhan2019 , I am your second reviewer, are you there so that we can have a little chat ? because I still can't see you inline reviews, of yourself.

**Comment by AsimKhan2019** [comment]
@ishimwezachee Yes I am here, I am in a morning session meeting and will be free shortly.

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Hi @AsimKhan2019 , as the first review asked, just implement the inline comments, and I guess the [videos ](https://www.loom.com/share/1d60a88c8ac543fe84764aaa7c51dc7a)I have made to guide you will help you. 


## Highlights


## Required Changes ♻️
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -12,7 +12,7 @@
 - Visual Studio Code and GitHub
 
 ## Live Demo (if available)
-https://htmlpreview.github.io/?https://github.com/AsimKhan2019/HTML_Mobile_Exercise/blob/main/index.html
+https://htmlpreview.github.io/?https://github.com/AsimKhan2019/video_youtube/blob/main/index.html
```
Comment (AsimKhan2019) [inline README.md:15]: - [ ] I think it is better to use some better formatting styles for the link

File: index.html [hunk 5]
```diff
@@ -10,14 +10,14 @@
 </head>
   
   <body>
-    <section id="main">
+    <section id="main"> <!-- defining main section here -->
       <header id="header">
```
Comment (AsimKhan2019) [inline index.html:14]: - [ ] You have applied the right approach using header tags instead of div tags.

File: style.css (hunk 3 above)
Comment (AsimKhan2019) [inline style.css:20]: - [ ] You have done well so far, now let's use numeric pixel for font size.

File: style.css [hunk 6]
```diff
@@ -30,7 +30,7 @@ body {
   padding-top: 0;
   padding-bottom: 0;
   border: 3px solid #808080;
-  border-radius: 8px;
+  border-radius: 7px;
   width: 590px;
```
Comment (AsimKhan2019) [inline style.css:34]: - [ ] Try using percentages for width size, instead of pixels. It helps with the resolution.

**Review by AsimKhan2019 — COMMENTED** [review]
You have done a great job. There are a few changes still needed. Have them reviewed, and keep up the good work. 

Cheers!

**Review by muneebulrehman — APPROVED** [review]
Hi @AsimKhan2019 

Great job on figuring out how to use inline comments. You would need them more than you think. Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@muneebulrehman Thank you for the feedback.

**Pull request merged**